fastlane/Fastfile

```diff
@@ -4,7 +4,7 @@ default_platform :ios
 require 'json'
 
 # The number of times the stress test suite is ran
-stress_tests_cycles = 10
+stress_tests_cycles = 5
```

Comment:
Instead of this, can you please change the `StressTestCase` number of cycles to say 10 + make the tests run randomly + increase this to ~50?

Basically, more cycles with random tests but less reparations of each tests

Reply:
What do you think about converting the `STRESS_TESTS` environment variable to something like `TEST_INVOCATIONS` so eventually we are able to set number of invocation from each CI job?

Reply:
I like it! I actually thought about running it at least once 100x and this would solve it perfectly 🎉